CondFormats/SiPixelTransient/src/SiPixelTemplate.cc

```diff
@@ -1328,6 +1328,12 @@ void SiPixelTemplate::postInit(std::vector<SiPixelTemplateStore>& thePixelTemp_)
 bool SiPixelTemplate::interpolate(int id, float cotalpha, float cotbeta, float locBz, float locBx) {
   // Interpolate for a new set of track angles
 
+  //check for nan's
+  if (!edm::isFinite(cotalpha) || !edm::isFinite(cotbeta)) {
```

Comment:
I think you want to maintain in the clause `#ifndef SI_PIXEL_TEMPLATE_STANDALONE` here, right?

Reply:
That's how the check was done before, and it is therefore the minimal fix.

Reply:
I spoke to Morris, he says I can add it for the standalone too.

Reply:
Why was not done like that at https://github.com/cms-sw/cmssw/pull/34846 ?
Please clarify

Reply:
Oz didnt ask Morris, I did... That's just it :)